src/Sulu/Bundle/AdminBundle/Resources/js/components/MultiSelect/MultiSelect.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import React from 'react';
+import type {Element} from 'react';
+import type {SelectChildren} from '../Select/types';
+import {GenericSelect, Option} from '../Select';
+
+type Props = {
+    values: Array<string>,
+    label: string,
+    onChange: (values: Array<string>) => void,
+    children: SelectChildren,
+    icon?: string,
+};
+
+export default class MultiSelect extends React.PureComponent<Props> {
+    static defaultProps = {
+        values: [],
+    };
+
+    optionIsSelected = (option: Element<typeof Option>): boolean => {
+        return this.props.values.indexOf(option.props.value) !== -1;
+    };
+
+    handleSelect = (value: string) => {
+        let newValues = [...this.props.values];
+        const index = newValues.indexOf(value);
+        if (index === -1) {
+            newValues.push(value);
+        } else {
+            newValues.splice(index, 1);
+        }
+        this.props.onChange(newValues);
+    };
+
+    getLabelText = () => this.props.label;
+
+    render() {
+        return (
+            <GenericSelect
+                icon={this.props.icon}
+                onSelect={this.handleSelect}
+                closeOnSelect={false}
```

Comment:
Because the `Label` component determines whether the select is open or not and that component is invisible to the user of the select I recommend not propagating the `open` state outwards of the component.
We had the `isOpen` mechanism in the overlay. However, in that case the mechanism on how the overlay is opened is invisible to the overlay and not the other way around. The material-ui framework follow the same strategy (with the `isOpen`) reagrding their dialogs. On their select component they also don't propagate the open state outwards.